pkg/network/tracer/connection/tracer.go

```diff
@@ -81,23 +81,23 @@ const (
 
 var ConnTracerTelemetry = struct {
 	connections       telemetry.Gauge
-	tcpFailedConnects telemetry.Gauge
-	TcpSentMiscounts  *nettelemetry.StatGaugeWrapper
-	missedTcpClose    telemetry.Gauge
-	missedUdpClose    telemetry.Gauge
-	UdpSendsProcessed *nettelemetry.StatGaugeWrapper
-	UdpSendsMissed    *nettelemetry.StatGaugeWrapper
-	UdpDroppedConns   telemetry.Gauge
+	tcpFailedConnects *nettelemetry.StatCounterWrapper
+	TcpSentMiscounts  *nettelemetry.StatCounterWrapper
+	missedTcpClose    *nettelemetry.StatCounterWrapper
+	missedUdpClose    *nettelemetry.StatCounterWrapper
+	UdpSendsProcessed *nettelemetry.StatCounterWrapper
+	UdpSendsMissed    *nettelemetry.StatCounterWrapper
+	UdpDroppedConns   *nettelemetry.StatCounterWrapper
 	PidCollisions     *nettelemetry.StatCounterWrapper
 }{
 	telemetry.NewGauge(connTracerModuleName, "connections", []string{"ip_proto", "family"}, "Gauge measuring the number of active connections in the EBPF map"),
-	telemetry.NewGauge(connTracerModuleName, "tcp_failed_connects", []string{}, "Gauge measuring the number of failed TCP connections in the EBPF map"),
-	nettelemetry.NewStatGaugeWrapper(connTracerModuleName, "tcp_sent_miscounts", []string{}, "Gauge measuring the number of miscounted tcp sends in the EBPF map"),
-	telemetry.NewGauge(connTracerModuleName, "missed_tcp_close", []string{}, "Gauge measuring the number of missed TCP close events in the EBPF map"),
-	telemetry.NewGauge(connTracerModuleName, "missed_udp_close", []string{}, "Gauge measuring the number of missed UDP close events in the EBPF map"),
-	nettelemetry.NewStatGaugeWrapper(connTracerModuleName, "udp_sends_processed", []string{}, "Gauge measuring the number of processed UDP sends in EBPF"),
-	nettelemetry.NewStatGaugeWrapper(connTracerModuleName, "udp_sends_missed", []string{}, "Gauge measuring failures to process UDP sends in EBPF"),
-	telemetry.NewGauge(connTracerModuleName, "udp_dropped_conns", []string{}, "Gauge measuring the number of dropped UDP connections in the EBPF map"),
+	nettelemetry.NewStatCounterWrapper(connTracerModuleName, "tcp_failed_connects", []string{}, "Gauge measuring the number of failed TCP connections in the EBPF map"),
```

Comment:
Descriptions should be updated to say "counter" instead of "gauge".

Reply:
blerg